src/OmniSharp.Roslyn.CSharp/Services/RoslynGotoDefinition.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using System.Collections.Generic;
+using System.Composition;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.FindSymbols;
+using Microsoft.CodeAnalysis.Text;
+using OmniSharp.Endpoints;
+using OmniSharp.Models;
+
+namespace OmniSharp.Roslyn.CSharp.Services
+{
+    [Export(typeof(IGotoDefintion))]
+    class RoslynGotoDefinition : IGotoDefintion
+    {
+        public Task<bool> IsApplicableTo(GotoDefinitionRequest request)
+        {
+            return Task.FromResult(OmnisharpWorkspace.Instance.GetDocument(request.FileName) != null);
```

Comment:
I thought about `return extension.Equals("cs")` but then `ScriptCs` and `Cake` crossed my mind, which would also be using the Roslyn Services.